lib/PipelineC/PipelineC.cpp

```diff
@@ -1110,32 +1116,37 @@ const rp_kind *rp_kind_get_preferred_kind(rp_kind *kind, uint64_t index) {
     return nullptr;
 }
 
-uint64_t rp_buffer_size(const rp_buffer *buffer) {
+static uint64_t _rp_buffer_size(const rp_buffer *buffer) {
   return buffer->size();
 }
 
-const char *rp_buffer_data(const rp_buffer *buffer) {
+static const char *_rp_buffer_data(const rp_buffer *buffer) {
   return buffer->data();
 }
 
-void rp_buffer_destroy(const rp_buffer *buffer) {
+static void _rp_buffer_destroy(rp_buffer *buffer) {
   delete buffer;
 }
 
-rp_container_targets_map *rp_container_targets_map_create() {
+static rp_container_targets_map *_rp_container_targets_map_create() {
   return new ContainerToTargetsMap();
 }
 
-void rp_container_targets_map_destroy(rp_container_targets_map *map) {
+static void _rp_container_targets_map_destroy(rp_container_targets_map *map) {
   revng_check(map != nullptr);
   delete map;
 }
 
-void rp_container_targets_map_add(rp_container_targets_map *map,
-                                  const rp_container *container,
-                                  const rp_target *target) {
+static void _rp_container_targets_map_add(rp_container_targets_map *map,
+                                          const rp_container *container,
+                                          const rp_target *target) {
   revng_check(map != nullptr);
   revng_check(container != nullptr);
   revng_check(target != nullptr);
   (*map)[container->second->name()].push_back(*target);
 }
+
+// clang-format off
+#include "Tracing/Wrapper.h"
+#include "revng/PipelineC/Wrappers.h"
```

Comment:
1. Add a comment describing what this does.
2. On a second thought, `Wrappers.h` contains function *definitions*, it's not really a `.h` file... I was trying to follow QEMU's convnetion on naming generated files, but nothing seems to fit well here. Let's leave it as `.h`.